ash/src/vk/bitflags.rs

```diff
@@ -252,12 +252,6 @@ impl FenceCreateFlags {
 }
 #[repr(transparent)]
 #[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
-#[doc = "<https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/VkSemaphoreCreateFlagBits.html>"]
-pub struct SemaphoreCreateFlags(pub(crate) Flags);
-vk_bitflags_wrapped!(SemaphoreCreateFlags, Flags);
-impl SemaphoreCreateFlags {}
```

Comment:
Note that these silly things are moving to `definitions.rs` because we're skipping the `SemaphoreCreateFlagBits` type now (not in a `<require>`) and then only later coming across `SemaphoreCreateFlags` which is handled by an _almost identical_ codepath (this is why we have the `const_cache` to deduplicate them): :vomiting_face:.

Reply:
This was caused by the addition of:

https://github.com/ash-rs/ash/blob/57de82d5f842b56a79060b9716ea98b9c0c3f2b8/generator/src/lib.rs#L2777-L2782

But now simply averted by using `.replace("FlagBits", "Flags")` :nauseated_face: